modules/qtwidgets/properties/collapsiblegroupboxwidgetqt.cpp

```diff
@@ -393,6 +346,25 @@ void CollapsibleGroupBoxWidgetQt::onSetVisible(Property* property, bool visible)
     if (property == property_) {
         PropertyWidgetQt::onSetVisible(property, visible);
     }
+
+    if (isChildRemovable()) {
+        for (auto&& elem : util::zip(properties_, propertyWidgets_)) {
+            if (get<0>(elem) == property) {
```

Comment:
You can use elem.first() or elem.second()

Reply:
then one should potentially update the examples for `util::zip` as well...

Reply:
True

Reply:
fixed and docs extended